tests/cloudpickle_test.py

```diff
@@ -613,8 +613,8 @@ def test_cell_manipulation(self):
             msg='cell contents not set correctly',
         )
 
-    def test_logger(self):
-        logger = logging.getLogger('cloudpickle.dummy_test_logger')
+    def check_logger(self, name):
```

Comment:
I tried to use pytest.mark.parametrize but it seems like I could not make it work with test classes deriving from `unittest.TestCase` (see this [SO answer](https://stackoverflow.com/a/18187731) which seems to indicate that this is not possible)